src/clr-angular/forms/combobox/model/single-select-combobox.model.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { ComboboxModel } from './combobox.model';
+
+export class SingleSelectComboboxModel<T> implements ComboboxModel<T> {
```

Comment:
I know this has been a huge project but I would like to see spec files for the model functionality like we have for the tree node component models.

Reply:
It was the first spec I added after this PR was launched :) 
Done.